### PR TITLE
Config for kernel development

### PR DESCRIPTION
**Pull request by TheMeaningfulEngineer**

### CODE REVIEW

File: arch/arm/configs/senic_defconfig_dev [hunk 1]
```diff
@@ -0,0 +1,3492 @@
...
+# CONFIG_ARCH_REALVIEW is not set
+# CONFIG_ARCH_ROCKCHIP is not set
+# CONFIG_ARCH_SOCFPGA is not set
+# CONFIG_PLAT_SPEAR is not set
+# CONFIG_ARCH_STI is not set
+# CONFIG_ARCH_S5PV210 is not set
+# CONFIG_ARCH_EXYNOS is not set
+# CONFIG_ARCH_RENESAS is not set
+CONFIG_ARCH_SUNXI=y
+CONFIG_MACH_SUN4I=y
```
Comment (baali) [inline arch/arm/configs/senic_defconfig_dev]: Would we need all of these derivatives? https://github.com/getsenic/senic-os-linux/blob/senic/4.10/arch/arm/configs/senic_defconfig#L8 is using just one and xi covers complete family as mentioned here: http://linux-sunxi.org/Main_Page

Comment (TheMeaningfulEngineer) [reply arch/arm/configs/senic_defconfig_dev]: A, sorry, most of the parameters were inherited from the original config file.
The pull request just gives a new file.

To get the idea what I actually changed check the difference between:
https://github.com/getsenic/senic-os-linux/blob/senic/4.10/arch/arm/configs/senic_defconfig
https://github.com/TheMeaningfulEngineer/senic-os-linux/blob/kernel-dev/arch/arm/configs/senic_defconfig_dev

Comment (baali) [reply arch/arm/configs/senic_defconfig_dev]: Yeah, I looked at the diff of these two files. For wireless support, I noticed config, `CONFIG_WIRELESS=y` which are missing from senic_defconfig.

Comment (TheMeaningfulEngineer) [reply arch/arm/configs/senic_defconfig_dev]: Don't know about that. 
I've edited the config file with `menuconfig` not by manually changing the file. It's possible it changed some required dependencies on its own.

The essence of this whole pull request is just some `m` switched to `y`

**Comment by TheMeaningfulEngineer** [comment]
@baali Am closing this pull request. It makes no more sense after the sync we had on kernel development

**Pull request closed without merging**